nix/overlay.nix

```diff
@@ -106,6 +106,12 @@ in
     # https://github.com/NixOS/nixpkgs/issues/130963
     NIX_LDFLAGS = lib.optionalString stdenv.isDarwin "-lc++abi";
   });
+  vastStatic = final.pkgsStatic.vast.override {
+    withPlugins = [
+      ../plugins/broker
+      ../plugins/pcap
+    ];
+  };
```

Comment:
Why would this manually need to list our plugins? That sounds like a repetition that should not be necessary to me.

Reply:
It has a plugin problem that needs to be fixed for Non-static. So I was using this code in our tests. Yes, the best place should be in nixos-tests. If the problem is resolved, it can be removed in the next commit.